jax_md/quantity.py

```diff
@@ -285,6 +285,39 @@ def is_integer(x: Array) -> bool:
   return x.dtype == jnp.int32 or x.dtype == jnp.int64
 
 
+def average_pair_correlation_results(gofr, species=None):
```

Comment:
This seems reasonable, but we could also add a keyword argument to `pair_correction` that says "compute_average` or something? Is there a reason you went with this approach?

To be honest, it could be nice to have both. Keep this function and then add a convenience keyword argument that averages after computing.

Reply:
I was actually thinking along the same lines. Just added it. Let me know if you have any other comments.